phpBB/styles/prosilver/template/posting_buttons.html

```diff
@@ -105,6 +105,16 @@
 	<button type="button" class="button button-icon-only bbcode-color" name="bbpalette" id="bbpalette" value="{L_FONT_COLOR}" onclick="change_palette();" title="{L_BBCODE_S_HELP}">
 		<i class="icon fa-tint fa-fw" aria-hidden="true"></i>
 	</button>
+	<!-- IF S_SMILIES_ALLOWED -->
+	<button type="button" class="button button-icon-only smiley_button" accesskey="y" name="addsmiley" title="{L_SMILEYS}">
```

Comment:
without 
 .smiley_button {
	white-space: normal;
}
dropdown looks like this.
![image](https://cloud.githubusercontent.com/assets/12277453/25318189/47806e66-28a7-11e7-98a0-2a6e920030d4.png)